manager/gui/src/components/object-store-config/LrTenant.vue

```diff
@@ -0,0 +1,82 @@
+<!--
```

Comment:
LrTenant.vue to Tenant.vue.

Reply:
Removed the Lr from prefix of all the files.